include/Dialect/Secret/IR/SecretDialect.td

```diff
@@ -2,11 +2,13 @@
 #define HEIR_INCLUDE_DIALECT_SECRET_IR_SECRETDIALECT_TD_
 
 include "mlir/IR/DialectBase.td"
+include "mlir/IR/OpBase.td"
 
 def Secret_Dialect : Dialect {
   let name = "secret";
-  let summary = "A dialect for computations that operate on encrypted secrets.";
```

Comment:
Should we note somewhere in a style guide not to use `summary`? Was it making docs look bad?

Reply:
I don't think it's bad, but it just emits the summary as an extra line above the description with no extra line break. So if you expect that then you can make it look right.